source/common/formatter/substitution_formatter.h

```diff
@@ -31,7 +31,7 @@ class SubstitutionFormatParser {
    * Parse a header format rule of the form: %REQ(X?Y):Z% .
    * Will populate a main_header and an optional alternative header if specified.
    * See doc:
-   * docs/root/configuration/access_log.rst#format-rules
+   * docs/root/configuration/observability/access_log/access_log.rst#format-rules
```

Comment:
i think these couple of links can also be updated to full urls

Reply:
Ok. Also made the no www prefix consistent. Let me know if anything else needs to be addressed.

Reply:
Just remembered that you requested it should go in another pull request, so just took that out now.